OracleSiebel/OEL/build/installerServer/readme.md

```diff
@@ -0,0 +1,35 @@
+# installerServer
+
+Before launching the server, you should first collect the required installation media. You will need:
+
+* Oracle DB Instant Client for 12.2.0.1
```

Comment:
The correct name of the product is "Oracle Instant Client". Please remove the "DB".

Reply:
Done